src/app/background/services/createContributionEmail.ts

```diff
@@ -4,7 +4,7 @@ import {
   formatContributionTextEmail
 } from 'app/lmem/format/contribution';
 import { truncateWords } from '../../utils/truncate';
-import { TransactionEmail } from 'SendInBlue';
+import { TransactionEmail } from '../../../api/sendInBlue/types';
```

Comment:
Same comment as mail related files that shouldn't be included in the profiles, thus not impacted by these changes.